src/test/kotlin/org/rustSlowTests/CargoProjectStructureTest.kt

```diff
@@ -91,6 +93,14 @@ class CargoProjectStructureTest : RsWithToolchainTestBase() {
             testRootDisposable,
             project.cargoProjects.allProjects.toList()
         )
-        ProjectViewTestUtil.assertStructureEqual(structure, expectedTreeStructure.trimIndent() + "\n", null)
+        assertStructureEqual(structure, expectedTreeStructure.trimIndent() + "\n")
+    }
+
+    private fun assertStructureEqual(structure: AbstractTreeStructure, expected: String) {
+        checkGetParentConsistency(structure, structure.rootElement)
+        val actual = PlatformTestUtil.print(structure, structure.rootElement) {
+            if (it is CargoProjectTreeStructure.CargoSimpleNode) it.toTestString() else it.toString()
+        }
+        assertEquals(expected, actual)
```

Comment:
Let's add comment that with code does the same as `com.intellij.testFramework.ProjectViewTestUtil#assertStructureEqual` but with custom node rendering

Reply:
Fixed.